src/modules/esl-avatar/core/esl-avatar.less

```diff
@@ -0,0 +1,35 @@
+esl-avatar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: opacity 0.3s ease-in-out;
+  opacity: 0;
+  border-radius: 50%;
+  background: var(--esl-avatar-background);
```

Comment:
Hm do we need to introduce custom variable for existing CSS property that used once (or can be inherited)...
PS: does not blocks current PR just a point to discuss for final epic

Reply:
removed